### PR TITLE
chore: bump dependencies

### PR DESCRIPTION
**Pull request by smira**
PRs:

* #4215
* #4216
* #4217
* #4218
* #4219
* #4220
* #4221

+ go-mod-outdated

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -41,30 +41,30 @@ require (
 	github.com/firecracker-microvm/firecracker-go-sdk v0.22.0
 	github.com/fsnotify/fsnotify v1.5.1
 	github.com/fullsailor/pkcs7 v0.0.0-20190404230743-d7302db945fa
-	github.com/gdamore/tcell/v2 v2.4.0
+	github.com/gdamore/tcell/v2 v2.4.1-0.20210905002822-f057f0a857a1
```
Comment (AlekSi) [inline go.mod:44]: Any reason to switch to non-tagged version?

Comment (smira) [reply go.mod:44]: no reason for me, it is coming via another direct dependency (via tview)

**Review by AlekSi — APPROVED** [review]

**Review by Unix4ever — APPROVED** [review]

**Pull request merged**